integration-tests/data/snaps/basic-desktop/meta/gui/echo.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Echo
+Comment=It echos stuff
+Exec=basic-desktop.echo
+Icon=${SNAP}/meta/gui/icon.png
+Terminal=true
+Type=Application
+Categories=Utilities;Useless;
+Keywords=echo;echo;
+StartupNotify=false
```

Comment:
Since that's easy, can we have an unsupported entry here just to make sure it's dropping those and passing through our validator instead of just copying it over blindly? Not(Contains) would be enough in the test.